ark/utils/notebooks_test.py

```diff
@@ -23,3 +37,234 @@ def test_example_spatial_analysis():
 
 def test_example_neighborhood_analysis():
     _exec_notebook('example_neighborhood_analysis_script.ipynb')
+
+
+# testing specific inputs for Segment_Image_Data
+def segment_notebook_setup(tb, deepcell_tiff_dir="test_tiff", deepcell_input_dir="test_input",
```

Comment:
This looks great. I think for a first pass, we should move all of the helper functions into a separate file. Then this test file will just have the code where we run these tests.